helix-term/src/keymap.rs

```diff
@@ -313,6 +372,70 @@ pub enum KeymapResult {
     Cancelled(Vec<KeyEvent>),
 }
 
+#[derive(Debug, Clone, Default, PartialEq, Eq, PartialOrd, Ord)]
+pub struct Binding {
+    pub keys: Vec<KeyEvent>,
+    pub is_fallback: bool,
+}
```

Comment:
Should we do an enum for this rather than having `is_fallback`?

Reply:
I can see the "logical" appeal, but I don't think it'd really accomplish much other than making the usage a little more annoying.